ackhandler/sent_packet_handler.go

```diff
@@ -333,7 +333,11 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 			h.bytesInFlight,
 			h.congestion.GetCongestionWindow())
 	}
-	return !(congestionLimited || maxTrackedLimited)
+	// Workaround for #555:
+	// Always allow sending of retransmissions. This should probably be limited
+	// to RTOs, but we currently don't have a nice way of distinguishing them.
```

Comment:
Well, it does in a way. RTXs still count towards the bytesInFlight, so subsequent "normal" TXs will be delayed accordingly. But: I'm trying to fix incorrect behavior here, and I'd be happy taking the perf hit for now.

But again, I'm fine just killing this PR until we have a proper fix (which might take a few weeks).